eessi_bot_job_manager.py

```diff
@@ -27,6 +27,7 @@
 import re
 import subprocess
 import time
+import os.path
```

Comment:
Just curious, do we need to import `os.path`? `os` is imported already (a few lines above).

Reply:
Then no, just `import os` is sufficient to use `os.path`